src/Plugin.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Migrations;
+
+use Cake\Core\BasePlugin;
+
+/**
+ * Plugin class for migrations
+ */
+class Plugin extends BasePlugin
+{
+    /**
+     * Plugin name.
+     *
+     * @var string
+     */
+    protected $name = 'Migrations';
+
+    /**
+     * Override to do nothing.
+     *
+     * @param \Cake\Routing\RouteBuilder $routes The route builder to update.
+     * @return void
+     */
+    public function routes($routes)
```

Comment:
Overriding the method is not necessary, instead you can set `$routesEnabled` property to false.